bin/start-systemtest

```diff
@@ -0,0 +1,206 @@
+#!/bin/bash
+
+##
+
+## checking args
+if [ $# -ne 3 ]
+then
+  echo "Usage: $0 USER_EMAIL USER_PASSWORD DISPATCHER_ADDRESS"
+  exit 1
+fi
+
+# args
+readonly USER_EMAIL=${1}
+readonly USER_PASSWORD=${2}
+readonly DISPATCHER_ADDRESS=${3}
+
+readonly SANDBOX=$(cd "$(dirname "${0}")" && pwd)
+readonly SUBMIT_TASK_SCRIPT=${SANDBOX}/submit-task
+readonly GET_TASK_BY_ID_SCRIPT=${SANDBOX}/get-task-by-id
+readonly GET_TASK_LINKS_SCRIPT=${SANDBOX}/get-task-links
+readonly SYSTEMTEST_MD5SUM_RESULTS_FILE=${SANDBOX}/../resources/systemtest-md5sum-results
```

Comment:
is there any directory whose path is ${SANDBOX}/../? It seems weird. ../ usually refers to the previous directory...

Reply:
This is just to go to the previous directory, but this ${SANDBOX} only guarantees that the starting point is the folder in which the script 'start-systemtest' is contained